src/Core/Annotation/Condition.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Contains Drupal\rules\Core\Annotation\Condition.
+ */
+
+namespace Drupal\rules\Core\Annotation;
+
+use Drupal\Core\Condition\Annotation\Condition as CoreConditionAnnotation;
+
+/**
+ * Extension of the Condition annotation class.
+ *
+ * @Annotation
+ *
+ * This class adds a configuration access parameter to the Condition
+ * annotation.
+ */
+class Condition extends CoreConditionAnnotation {
+
+  /**
+   * The permission required to access the configuration UI for this plugin.
+   *
+   * @var string
+   *   Permission string as declared in a *.permissions.yml file.
+   */
+  public $configuration_access;
```

Comment:
Should be $configure_permission and must allow array values also + document how that works.